src/realm/backup_restore.cpp

```diff
@@ -0,0 +1,172 @@
+/*************************************************************************
+ *
+ * Copyright 2016 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ **************************************************************************/
+
+#include <realm/backup_restore.hpp>
+#include <realm/util/file.hpp>
+
+#include <vector>
+#include <chrono>
+
+namespace realm {
+
+/*
+ * IMPORTANT: The following two arrays must be kept updated
```

Comment:
This documentation belongs in the header with the original declarations. Also, if there is a requirement that they be sorted (which they currently are), that should also be stated.